docs/5.0/preview/database-access.md

```diff
@@ -149,34 +230,70 @@ ssh_service:
 Start the service:
 
 ```sh
-$ teleport start --debug --config=/path/to/teleport.yaml
+$ teleport start -d --config=/path/to/teleport.yaml
 ```
 
-### Start Teleport Database Service
+### Start Database Service with CLI Flags
+
+For a quick try-out, Teleport database service doesn't require a configuration
+file and can be launched using a single CLI command:
+
+```sh
+$ teleport start -d \
+   --roles=database \
+   --token=cbdeeab9-6f88-436d-a673-44d14bd86bb7 \
+   --auth-server=teleport.example.com:3080 \
+   --db-name=test \
+   --db-protocol=postgres \
+   --db-uri=db.example.com:5432 \
+   --labels=env=test
+```
 
-Create configuration file for a database service:
+Note that the `--auth-server` flag must point to cluster's proxy endpoint
+because database service always connects back to the cluster over a reverse
+tunnel.
+
+Instead of using a static auth token, a short-lived dynamic token can also
+be generated for a database service:
+
+```sh
+$ tctl tokens add \
+    --type=database \
+    --db-name=test \
+    --db-protocol=postgres \
+    --db-uri=db.example.com:5432
```

Comment:
nit: can we use an example output to hint at what a dynamic token could look like?

Reply:
Not sure it'd add a lot of value to the reader to be honest, outputs also have a tendency to change / become obsolete.